connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -485,16 +486,40 @@ public static synchronized void sdkReinitializeMobileConnect() {
         profile.deInitialize();
     }
 
-    public static ConnectivityManager getConnectivityManager() {
-        return connectivityManager;
+    /**
+     * Initialize components common to both Mobile Connect and ConnectID SDK profiles
+     */
+    public static synchronized void initializeCommonComponents() {
```

Comment:
Hmm... I hadn't a big (killer) reason, apart from wanting to keep the initialise methods kinda together, but it's a matter of taste.